packages/clerk-js/src/ui/components/UserProfile/__tests__/ConnectedAccountsSection.test.tsx

```diff
@@ -1,50 +1,82 @@
 import type { ExternalAccountResource } from '@clerk/types';
 import { describe, it } from '@jest/globals';
-import React from 'react';
+import { act, waitFor } from '@testing-library/react';
 
 import { render, screen } from '../../../../testUtils';
 import { bindCreateFixtures } from '../../../utils/test/createFixtures';
 import { ConnectedAccountsSection } from '../ConnectedAccountsSection';
 
 const { createFixtures } = bindCreateFixtures('UserProfile');
 
-const initConfig = createFixtures.config(f => {
+const withoutConnections = createFixtures.config(f => {
   f.withSocialProvider({ provider: 'google' });
+  f.withSocialProvider({ provider: 'github' });
   f.withUser({});
 });
 
-//TODO-RETHEME
-describe.skip('ConnectedAccountsSection ', () => {
+const withSomeConnections = createFixtures.config(f => {
+  f.withSocialProvider({ provider: 'google' });
+  f.withSocialProvider({ provider: 'github' });
+  f.withUser({
+    external_accounts: [{ provider: 'google', email_address: 'test@clerk.com' }],
+  });
+});
+
+const withConnections = createFixtures.config(f => {
+  f.withSocialProvider({ provider: 'google' });
+  f.withSocialProvider({ provider: 'github' });
+  f.withUser({
+    external_accounts: [
+      { provider: 'google', email_address: 'test@clerk.com' },
+      { provider: 'github', email_address: 'test@clerk.com' },
+    ],
+  });
+});
+
+describe('ConnectedAccountsSection ', () => {
   it('renders the component', async () => {
-    const { wrapper } = await createFixtures(initConfig);
+    const { wrapper } = await createFixtures(withoutConnections);
 
-    render(<ConnectedAccountsSection />, { wrapper });
+    const { getByText, getByRole } = render(<ConnectedAccountsSection />, { wrapper });
+
+    getByText(/^Connected accounts/i);
+    getByRole('button', { name: /connect account/i });
   });
 
-  it('shows the title', async () => {
-    const { wrapper } = await createFixtures(initConfig);
+  it('renders the component with some enabled connections', async () => {
+    const { wrapper } = await createFixtures(withSomeConnections);
 
-    render(<ConnectedAccountsSection />, { wrapper });
+    const { getByText, getByRole } = render(<ConnectedAccountsSection />, { wrapper });
 
-    screen.getByRole('heading', { name: /add connected account/i });
+    getByText(/^Connected accounts/i);
+    getByText(/google/i);
+    getByText(/test@clerk.com/i);
+    getByRole('button', { name: /connect account/i });
```

Comment:
❓  I  would expect to see assertions here instead of failures caused by raised errors from `getBy*`. Using `getBy*` makes the expectation of the test implicit.

cc: @nikosdouvlis , @panteliselef